ChildProject/projects.py

```diff
@@ -290,7 +296,7 @@ def accumulate_metadata(
         merge_column: str,
         verbose=False,
     ) -> pd.DataFrame:
-        md_path = os.path.join(self.path, "metadata", table)
+        md_path = os.path.join(self.path, METADATA_FOLDER, table)
```

Comment:
ok this resolves to the same thing, since md_folder defined above - any reason to make it into a variable? should we do the same elsewhere?

Reply:
We should definitely do this as much as we can. This brings more flexibility in changing the code like explained above